Documentation/compatibility/wpf-accessibility-improvements.MD

```diff
@@ -35,19 +35,19 @@ __Screen reader interaction improvements__
 __LiveRegion support__   
 
 Screen readers such as Narrator help people know the UI contents of an application, usually by describing something about the UI that's currently focused, because that is probably the element of most interest to the user. However, if a UI element changes somewhere in the screen and it does not have the focus, the user may not be informed and miss important information.
-LiveRegions are meant to solve this problem. A developer can use them to inform the screen reader or any other [UI Automation][UI Automation Overview](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-overview) client that an important change has been made to a UI element. The screen reader can then decide how and when to inform the user of this change.
+LiveRegions are meant to solve this problem. A developer can use them to inform the screen reader or any other [UI Automation](~/docs/framework/ui-automation/ui-automation-overview.md) client that an important change has been made to a UI element. The screen reader can then decide how and when to inform the user of this change.
 The LiveSetting property also lets the screen reader know how important it is to inform the user of the change made to the UI.
 
 
 - [x] Quirked
 - [ ] Build-time break
 
 ### Recommended Action
-__How to opt in or out of these changes__
+__How to opt in or out of these changes__   
```

Comment:
Remove trailing space

Reply:
@conniey, I'd included the trailing spaces in the hope of adding a line break, though I'm not sure that that will work. Does the migration routine trim them? For a line break in the middle of a section, do we need to use an HTML \<\\p> tag?

Reply:
iirc, we trim all trailing whitespace as we are going line through line when reading the documentation, so adding trailing whitespace wouldn't help.  How the parser understands when to put a `<p>` is when there is an additional new line between the lines. The following example would render a `<p>`:
```
This is a line.

This is my new paragraph.
```

Whereas:
```
This is one line.
This is a second line.
```
Results in : "This is one line. This is a second line."
Is this how it works, currently, @chlowell ?

Reply:
I don't think that's how it works, @conniey. There are numerous cases in the original markdown files where there is an additional line between the lines, and there's no line break in the migrated markdown.

Reply:
Whitespace is trimmed away during parsing of the source document. The tool removes all newlines from the source when it writes an include file because each section becomes a markdown table row, which would be broken by a newline.

I don't know how `<p>` within a table row would render on the site. I think it's legal GFM. The migration tool doesn't write `<p>` elements, but any that are in the source documents will be transcribed verbatim into the output.

Reply:
Thanks, @chlowell. That's good to know. 
I'll remove the trailing spaces here where I added them and accept this PR. 
\</p> or \</br> should be fine in markdown tables. Just to be sure, I'll add some to one of the topics that I include in this PR. Then the current content should be ready for a new round of migration to the dotnet/docs repo, and we can see how the HTML tags affect the published content.